connectors/open_policy_agent/go/api_default.go

```diff
@@ -13,18 +13,54 @@
 package openapiserver
 
 import (
-	"io/ioutil"
 	"log"
 	"net/http"
 
+	opabl "fybrik.io/fybrik/connectors/open_policy_agent/lib"
+	openapiclientmodels "fybrik.io/fybrik/pkg/taxonomy/model/base"
+	"github.com/gdexlab/go-render/render"
 	"github.com/gin-gonic/gin"
 )
 
-// GetPoliciesDecisions - getPoliciesDecisions
-func GetPoliciesDecisions(c *gin.Context) {
+type ConnectorController struct {
+	Policytobeevaluated string
+	Opaserverurl        string
+	Opaclient           *http.Client
+	CatalogReader       *opabl.CatalogReader
+}
+
+func (e *ConnectorController) contactOPA(input openapiclientmodels.PolicyManagerRequest, creds string) (openapiclientmodels.PolicyManagerResponse, error) {
+	opaReader := opabl.NewOpaReader(e.Opaserverurl)
```

Comment:
Reuse opa client instead of creating a new one on each call

Reply:
fixed. Please check. Thanks.